### PR TITLE
Create aayush.json

### PR DESCRIPTION
**Pull request by OpAayush**

### CODE REVIEW

**Comment by OpAayush** [comment]
now is it fine?

**Comment by wclarkey** [comment]
@Tweak4141 Validation shows error that i cant personally see

File: sub-logs/aayush.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "owner": {
+        "username": "OpAayush",
+        "email": "aayushmagrawal@gmail.com"
+    },
+
+    "target": {
+      "CNAME": {
+            "name": "aayush",
+            "value": "opaayush.pages.dev"
+        }
```
Comment (WilliamDavidHarrison) [inline sub-logs/aayush.json:13]: Add a comma at the end of this line.

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]

**Comment by WilliamDavidHarrison** [comment]
> @Tweak4141 Validation shows error that i cant personally see

Comma missing

**Comment by OpAayush** [comment]
What about now.

File: sub-logs/aayush.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "website": "https://opaayush.pages.dev",
+
```
Comment (WilliamDavidHarrison) [inline sub-logs/aayush.json:3]: Remove this field

File: sub-logs/aayush.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+    "website": "https://opaayush.pages.dev",
```
Comment (WilliamDavidHarrison) [inline sub-logs/aayush.json:2]: Remove this field

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]

**Comment by Tweak4141** [comment]
@OpAayush Please fix your JSON issue or I can't merge this.

**Comment by OpAayush** [comment]
now it should be fine

**Review by WilliamDavidHarrison — APPROVED** [review]

**Pull request merged**